src/student_class.rb

```diff
@@ -3,10 +3,10 @@
 class Student < Person
   attr_reader :classroom
 
-  def initialize(age, name = 'Unknown', classroom = 'Unknown')
+  def initialize(age, name = 'Unknown', classroom = 'Unknown', _parent_permission: true)
```

Comment:
- [ ] There is a bug 🐛 when attempting to give a student `_parent_permission` while creating that student. Please noted that replacing 
```ruby 
_parent_permisson: true
```
with 
```ruby
_parent_permission = true
```
fixes this bug 😎